scanpy/neighbors/__init__.py

```diff
@@ -836,75 +589,48 @@ def compute_neighbors(
             self._connected_components = connected_components(self._connectivities)
             self._number_connected_components = self._connected_components[0]
 
-    def _compute_connectivities_diffmap(self, density_normalize=True):
-        # init distances
-        if self.knn:
-            Dsq = self._distances.power(2)
-            indices, distances_sq = _get_indices_distances_from_sparse_matrix(
-                Dsq, self.n_neighbors
+    @staticmethod
+    def _handle_transform_args(
+        method: _Method | Literal['gauss'],
+        transformer_cls: type | _KnownTransformer | None,
+        *,
+        knn: bool,
+    ) -> tuple[_Method, type, bool]:
```

Comment:
Could you document this method? A short description of behavior + what the output arguments mean – especially "shortcut".